Examples/Tests/ohm_solver/ion_Landau_damping/analysis.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+#
+# --- Analysis script for the hybrid-PIC example of ion Landau damping.
+
+import dill
+import matplotlib
+import matplotlib.pyplot as plt
+import numpy as np
+
+from pywarpx import picmi
+
+constants = picmi.constants
+
+matplotlib.rcParams.update({'font.size': 20})
+
+# load simulation parameters
+with open(f'sim_parameters.dpkl', 'rb') as f:
+    sim = dill.load(f)
+
+if sim.test:
```

Comment:
Same comment as for the other script: maybe we could put this at the end of the script and at the same time add a physics test that checks the damping rate with an `assert` statement.